### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

**Comment by parulgupta2001** [comment]
please review my PR

File: docs/avatar.html [hunk 1]
```diff
@@ -1 +1,93 @@
...
+        <div class="description">
+            <p1>Avatars are images that users can set as their profile picture.They can be custom photos, uploaded by users, or generated as Identicons as a placeholder.
+            </p1>
+        </div>
+
+
+        <div class="avatar-container-size">
+            <p>Avatars with different sizes</p>    
+            <div class="avatar-img">
+                <img class="avatar avatar-sm" src="/Images/1.jpg" alt="" >
```
Comment (dev-enforced) [inline docs/avatar.html:48]: Don't keep alt attribute of images as empty. Better practice to have alt related to image description for better accessibility.

File: styles/main-style.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    flex-direction: column;
+}
+
+.get-started-page-content>h3 {
+   padding-top: 0;
+   margin-top: 0;
+
+}
+
+.elucidate {
```
Comment (dev-enforced) [inline styles/main-style.css:116]: You can create a separate utility class named as ml-15 which has this property and pass it into that particular HTML element.

**Review by dev-enforced — CHANGES_REQUESTED** [review]
The output and code looks really good Parul. You can merge it. Attached some suggestions below for better understanding.

File: docs/avatar.html (hunk 1 above)
Comment (parulgupta2001) [reply docs/avatar.html:48]: > Don't keep alt attribute of images as empty. Better practice to have alt related to image description for better accessibility.

thanks for your suggestion, I will definitely do this

File: styles/main-style.css (hunk 2 above)
Comment (parulgupta2001) [reply styles/main-style.css:116]: > You can create a separate utility class named as ml-15 which has this property and pass it into that particular HTML element.

Its not working.

**Comment by therajatg** [comment]
Looks good to me Parul. Go ahead and merge it.

**Comment by parulgupta2001** [comment]
> Looks good to me Parul. Go ahead and merge it.

ok thanku

**Pull request merged**